src/ProjectBadge.js

```diff
@@ -1,25 +1,34 @@
 import React from "react";
 
+import abbreviationYAxisFn from "./abbreviationYAxisFn";
 import reducerFn from "./reducerFn";
-import {GlobalStyle, MiddleRing, InnerRing, Svg} from './ProjectBadge.styles'
+import { GlobalStyle, MiddleRing, InnerRing, Svg } from "./ProjectBadge.styles";
 
-const ProjectBadge = ({ color, abbreviation, description, number, className }) => {
+const ProjectBadge = ({
+  color = "#FF00FF",
+  abbreviation = "De",
```

Comment:
I wouldn't default on abbreviation, description or number -- it's better to blow up the component that display placeholder information.

If there's a graceful way to render a component without some of these, eg. number or description, then make it so. But if e.g. abbreviation is missing, there's not much you can do with this component.